plugins/modules/zos_mount.py

```diff
@@ -0,0 +1,1090 @@
+            automove=dict(
+                type='str',
+                default='AUTOMOVE',
+                choices=[
+                    'AUTOMOVE',
+                    'NOAUTOMOVE',
+                    'UNMOUNT'],
+                required=False),
+            automove_list=dict(type='str', required=False)
+        ),
+        add_file_common_args=True,
+        supports_check_mode=True
+    )
+
+    arg_def = dict(
+        src=dict(arg_type='data_set', required=True),
+        path=dict(arg_type='path', required=True),
+        fstype=dict(arg_type='str', choices=[
+                    "HFS", "zFS", "NFS", "TFS"], required=True),
+        state=dict(
+            arg_type='str',
+            default='mounted',
+            choices=[
+                'absent',
+                'mounted',
+                'unmounted',
+                'present',
+                'remounted'],
+            required=False),
+        fstab=dict(
+            arg_type='str',
+            default='',
+            required=False,
+            aliases=[
+                'bpxfile',
+                'bpxprm']),
+        backup=dict(arg_type='bool', default=False, required=False),
+        backup_name=dict(
+            arg_type='str',
+            default='',
+            required=False,
+            aliases=[
+                'backup_file',
+                'backup_bpxprm']),
+        tabcomment=dict(arg_type='list', elements='str', required=False),
+        unmount_opts=dict(
+            arg_type='str',
+            default='NORMAL',
+            choices=[
+                'DRAIN',
+                'FORCE',
+                'IMMEDIATE',
+                'NORMAL',
+                'REMOUNT',
+                'RESET'],
+            required=False),
+        mount_opts=dict(
+            arg_type='str',
+            default='rw',
+            choices=[
+                'ro',
+                'rw',
+                'same',
+                'nowait',
+                'nosecurity'],
+            required=False),
+        src_params=dict(arg_type='str', default='', required=False),
+        tag_flag=dict(
+            arg_type='str',
+            default='',
+            choices=[
+                '',
+                'TEXT',
+                'NOTEXT'],
+            required=False),
+        tag_ccsid=dict(arg_type='str', required=False),
+        allow_uids=dict(arg_type='bool', default=True, required=False),
+        sysname=dict(arg_type='str', default='', required=False),
+        automove=dict(
+            arg_type='str',
+            default='AUTOMOVE',
+            choices=[
+                'AUTOMOVE',
+                'NOAUTOMOVE',
+                'UNMOUNT'],
+            required=False),
+        automove_list=dict(arg_type='str', default='', required=False)
+    )
+
+    try:
+        res_args = None
+        res_args = run_module(module, arg_def)
+        module.exit_json(**res_args)
+    finally:
+        pass
+
```

Comment:
Is there a particular reason why you have a `try..finally` block here? Seems like a no-op. Not a big deal though.